pandas/core/series.py

```diff
@@ -1350,7 +1350,14 @@ def repeat(self, repeats, axis=None) -> Series:
         )
 
     @deprecate_nonkeyword_arguments(version=None, allowed_args=["self", "level"])
-    def reset_index(self, level=None, drop=False, name=lib.no_default, inplace=False):
+    def reset_index(
+        self,
+        level=None,
+        drop=False,
+        name=lib.no_default,
+        inplace=False,
+        allow_duplicates: bool = False,
```

Comment:
should be None right?

Reply:
I believe that the concensus it to _not_ change the existing behavior.
So, for Series, I want the default to be False.
For DataFrame there is an additional option and I have used None. This means to use the frame's own self.flags.allows_duplicate_labels.
In the end, they are calling DataFrame.insert, where the default is and remains False:

```
    def insert(
        self,
        loc: int,
        column: Hashable,
        value: Scalar | AnyArrayLike,
        allow_duplicates: bool = False,
    ) -> None:
        """
        Insert column into DataFrame at specified location.
```

Reply:
+1 on defaulting to False.

Reply:
see my comment above